script.js

```diff
@@ -6,13 +6,120 @@ const menuIcon = document.querySelector('.menuIcon');
 const menuItem = document.querySelector('.menu-item-desktop');
 const items = document.querySelectorAll('.menuItemDesktop');
 
+const body = document.querySelector('body');
+const cardData = [
+  {
+    name: 'Project 1',
+    description:
+      'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam nobis nemo vitae tempora et magni in corporis earum illum cupiditate repudiandae, amet nihil reprehenderit voluptatum tempore eligendi deserunt id adipisci.',
+    featuredimg: './assets/img/laptop.png',
+    technologies: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+    linklive: 'http://google.com',
+    linksrc: 'http://github.com',
+  },
+  {
+    name: 'Project 2',
+    description:
+      'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam nobis nemo vitae tempora et magni in corporis earum illum cupiditate repudiandae, amet nihil reprehenderit voluptatum tempore eligendi deserunt id adipisci.',
+    featuredimg: './assets/img/laptop 2.png',
+    technologies: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
```

Comment:
- [ ] Lastly, please style the middle button so it looks similar to the other two :+1: 
- Your design
![image](https://user-images.githubusercontent.com/102699958/232047467-9dc415cb-7ac8-4800-ba44-d4026c5529b5.png)

- The Figma design
![image](https://user-images.githubusercontent.com/102699958/232047613-130421fb-cc4e-4b39-bcc4-915de3726318.png)